edx_proctoring/static/proctoring/js/dropdown.js

```diff
@@ -0,0 +1,84 @@
+edx = edx || {};
```

Comment:
Is this a copied over file? If so, can you specify in the comments where this is copied from? Give a comment reference makes this an easier reference in the future.